boards/arm/msp_exp432e401y_dk/Kconfig

```diff
@@ -0,0 +1,8 @@
+# TI MSP-EXP432E401Y-DK configuration
+# Copyright (c) 2021 pipei.sandor@gmail.com
+
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_MSP_EXP432E401Y_DK
+
+endif # BOARD_MSP_EXP432E401Y_DK
```

Comment:
Remove this file, it doesn't do anything

Reply:
Ok. Wil be removed.